environments/values-roundtable-dev.yaml

```diff
@@ -2,6 +2,8 @@ name: roundtable-dev
 fqdn: roundtable-dev.lsst.cloud
 vaultUrl: "https://vault.lsst.codes"
 vaultPathPrefix: secret/k8s_operator/roundtable-dev.lsst.cloud
+onepassword:
+  vaultTitle: "RSP roundtable-dev.lsst.cloud"
```

Comment:
May as well add `connectUrl: https://roundtable-dev.lsst.cloud/1password` as well, since the server is up and running although not fully populated yet.